repaygithub/ui-tools
# Add loaders for CSS and FTL files to repay-scripts

Ticket: https://repayonline.atlassian.net/browse/CACTUS-261

### How to Test

To regression test, you could link this package into one of the Cactus example apps and try to run both the build and dev commands.

If you want to actually test that this now enables Cactus dev, I can send you a tiny app (I don't want to clutter GitHub with it) you can use to test.  You can then test it with the published version and this version to see that these updates now correctly process CSS and FTL.  You can also just spin up your own app if you want.
